src/main/java/com/techsophy/tsf/account/constants/AccountConstants.java

```diff
@@ -9,11 +9,16 @@
  //DatabaseChangeLog
  public static final String ORDER_1="1";
  public static final String ORDER_2="2";
+ public static final String ORDER_3="3";
```

Comment:
Define ORDER as three digit values, dont modify for old order.
Please discuss if you are not clear.